app/models/what_degree_class.rb

```diff
@@ -1,5 +1,15 @@
 class WhatDegreeClass < Base
   attribute :degree_class, :string
 
-  validates :degree_class, inclusion: { in: ["Not applicable", "First class", "2:1", "2:2", "Third class"], message: "You must select an option" }
+  validates :degree_class, types: { method: :get_qualification_uk_degree_grades, message: "You must select an option" }
+
+  def self.options
+    option_list = {}
```

Comment:
I think I saw this method elsewhere as well? is it worth having a generic method that converts an array of API types to options?

Reply:
Sure, we have 3 models with self.options, 2 have a duplicable method, have put it in an extendable module.

Reply:
can use this elsewhere also